Interfaces/Cosmos.DataTransfer.Common/DataItemJsonConverter.cs

```diff
@@ -111,6 +136,60 @@ private static JsonEncodedText GetAsUnescaped(string text)
         return JsonEncodedText.Encode(text, JavaScriptEncoder.UnsafeRelaxedJsonEscaping);
     }
 
+    internal static bool TryGetLong(object? x, out long number) 
+    {
+        if (x is long l) {
```

Comment:
Curious as to why not use the [Convert ](https://learn.microsoft.com/dotnet/api/system.convert?view=net-9.0) class?